code/modules/mob/living/carbon/human/species_types/plasmamen.dm

```diff
@@ -247,47 +247,53 @@
 	to_add += list(
 		list(
 			SPECIES_PERK_TYPE = SPECIES_POSITIVE_PERK,
-			SPECIES_PERK_ICON = "user-shield",
-			SPECIES_PERK_NAME = "Protected",
-			SPECIES_PERK_DESC = "Plasmamen are immune to radiation, poisons, and most diseases.",
+			SPECIES_PERK_ICON = "user-astronaut",
+			SPECIES_PERK_NAME = "The Suit",
+			SPECIES_PERK_DESC = "Plasmamen begin wearing an airtight envirosuit, which they are dependant on to stay alive. \
+								Their envirohelm is also space-worthy, provides welding protection, and a built-in headlamp.",
 		),
 		list(
 			SPECIES_PERK_TYPE = SPECIES_POSITIVE_PERK,
-			SPECIES_PERK_ICON = "bone",
-			SPECIES_PERK_NAME = "Wound Resistance",
-			SPECIES_PERK_DESC = "Plasmamen have higher tolerance for damage that would wound others.",
+			SPECIES_PERK_ICON = "skull",
+			SPECIES_PERK_NAME = "Hazard Born",
+			SPECIES_PERK_DESC = "Plasmamen are reborn by the plasmafire they died in. \
+								They're completely immune to husking, starving, the cold, and resist diseases.",
 		),
 		list(
 			SPECIES_PERK_TYPE = SPECIES_POSITIVE_PERK,
-			SPECIES_PERK_ICON = "wind",
-			SPECIES_PERK_NAME = "Plasma Healing",
-			SPECIES_PERK_DESC = "Plasmamen can heal wounds by consuming plasma.",
+			SPECIES_PERK_ICON = "bone",
+			SPECIES_PERK_NAME = "Bones of Steel",
+			SPECIES_PERK_DESC = "Plasmamen have tough bones infused with solid plasma. \
+								They only suffer fracture wounds, and are generally harder to wound.",
```

Comment:
Are they?
They dont have innate wound armour but i guess being immune to flesh wounds makes them harder to wound

Reply:
they do got `TRAIT_HARDLY_WOUNDED`
![image](https://github.com/yogstation13/Yogstation/assets/143908044/263ae6be-49c9-4fa4-9282-f67eb2f5bd57)

Reply:
oh, huh, never knew